Persistence.MongoDB/README.md

```diff
@@ -0,0 +1,17 @@
+﻿This project implements the [Persistence interface](../Persistence) using a MongoDB database.
+It is designed to seamlessly function with the same database the old core uses.
+
+## run MongoDB
+
+- Install MongoDB 4.2 or higher. Get it [here](https://www.mongodb.com/download-center/community).
+- The server must be in [replica set mode](https://docs.mongodb.com/manual/tutorial/convert-standalone-to-replica-set/).
```

Comment:
This is only required because transactions require the server to be in replica set mode. however, for development it might be just fine to not have multi-statement and multi-document transactions. However, this would require the code to detect this and do things differently. The connection string would be different, and it would branch at each transaction to just not do the transaction if the server is in standalone mode.

My gut feeling says a slightly more complicated one-time development system setup is the lesser evil here